src/main/java/org/reaktivity/nukleus/maven/plugin/internal/generate/ListFlyweightGenerator.java

```diff
@@ -144,6 +144,26 @@ private MethodSpec forEachMethod()
                 .build();
     }
 
+    private MethodSpec anyMatchMethod()
+    {
+        ClassName predicateRawType = ClassName.get(Predicate.class);
+        TypeName itemType = WildcardTypeName.supertypeOf(thisName.typeArguments.get(0));
+        TypeName consumerType = ParameterizedTypeName.get(predicateRawType, itemType);
+        return methodBuilder("anyMatch")
```

Comment:
Suggest naming this method `matchAny` to have the naming structure consistent with the standard `Stream.findAny` method.